mirte_telemetrix/scripts/pcb_mapping/stm32.py

```diff
@@ -0,0 +1,77 @@
+# Map to convert from Mirte PCB to STM32 pins numbers
+# This should be the same as printed on the PCB
+mirte_pcb_map = {
```

Comment:
This one is not needed. This is an old pcb. So we could have a folder with deprecated_pcb/pcb_v04 or so. But no need to have the mapping fo the names on the PCB.